src/core/control/Control.cpp

```diff
@@ -2871,6 +2884,17 @@ void Control::showAbout() {
     dlg.show(GTK_WINDOW(this->win->getWindow()));
 }
 
+auto Control::loadViewMode(size_t mode) -> bool {
```

Comment:
```suggestion
auto Control::loadViewMode(ViewModeId mode) -> bool {
```
And similarly for all other occurences

Reply:
Okay, I can do that